core/src/main/scala/com/softwaremill/kmq/redelivery/streams/CustomHolder.scala

```diff
@@ -0,0 +1,13 @@
+package com.softwaremill.kmq.redelivery.streams
+
+/**
+ * Custom mutable single-item collection.
+ * @tparam V value type
+ */
+class CustomHolder[V] {
```

Comment:
We definitely need better names for the `Custom*` stuff. I think what you have here is usually called a `Cell` or `Ref`

Reply:
Okay, changed names to:
* CustomHolder -> Ref
* CustomPriorityQueueMap -> PriorityQueueMap